__tests__/app.test.js

```diff
@@ -19,3 +19,37 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe("GET /api/reviews", () => {
+  test("should respond with 200 and get all reviews with an added key of comment count", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews[5]).toEqual({
+          review_id: 2,
+          title: "Jenga",
```

Comment:
Could you also check the length of the array, to make sure you're getting back all the reviews in the dataset?